src/actions/install-update.js

```diff
@@ -103,6 +106,12 @@ module.exports = function hydrator (params, callback) {
         exec(`bundle update`, options, callback)
       }
 
+      // cache deno deps.ts
+      else if (isDeno) {
+        // should --reload be added? That would force re-caching everytime, so maybe not?
```

Comment:
Hydrate is responsible for deterministic deploys, prob not a bad idea to use `--reload` but I'm not sure what the re-caching operations are in Deno-land – can you speak to that at all?

Reply:
Yeah, damn, bummer about the import-URLs-with-no-version. With that in mind I guess it makes sense to `--reload` all the time. As was said, we take the performance hit up-front at author time during `hydrate` (and thus before a `deploy` or `sandbox` run) in order to keep runtime performant.

Reply:
OK, sure. Latest commit takes that approach

Reply:
You'll see Deno's cache output in stdout, can add `--quiet` to the cache command to prevent that

Also, as we're caching the lambda handler, this doesn't quite follow the format of the logging
```
✓ Hydrate Successfully hydrated dependencies
⚬ Hydrate Hydrating dependencies in 1 path
✓ Hydrate Hydrated frontend/home//home/
```

`Hydrate Hydrated frontend/home//home/` should really be `Hydrate Hydrated frontend/home/index.tsx`

Think that uses dirname somewhere?